tests/installation/partitioning_olddisk.pm

```diff
@@ -0,0 +1,96 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary:  Install on old disk
+# Maintainer: Joyce Na <jna@suse.de>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use version_utils qw(is_storage_ng is_sle);
+use testapi;
+
+sub run() {
```

Comment:
This file is a little tightly designed. If it can provide the freedom to configure which disk, partition to have what fs and mount point, then it will be a quite good fundamental installation module. 
But it is fine to be what it is now as the first step.

Reply:
Yes,good suggestion,but may be next step.